### DISCUSSION TITLE
Searching for custom field value doesn't return draft entries

### DISCUSSION THREAD

**Discussion by benface** [craftcms/cms#9015] [Ideas]
### Steps to reproduce

1. Create a Plain Text field named "Test" and add it to some entry type's field layout.
2. Create a draft entry of that type, with "title 1" as the title and "test 1" in the "Test" field. Don't publish it.
3. Create another entry of the same type, with "title 2" as the title and "test 2" in the "Test" field. Publish this one.
4. In the entries index, search for "title". Notice both entries, the draft and the published one, show up in the results.
5. Still in the entries index, search for "test". Notice only the published entry appears in the results.

Expected result: Both entries should appear.

### Additional info

- Craft version: 3.6.11.1
- PHP version: 8.0.3
- Database driver & version: MySQL 8.0.23

**Comment by brandonkelly** [comment]
This is currently expected, as only the titles are indexed for drafts currently. Will treat this as an enhancement request.

**Comment by benface** [comment]
@brandonkelly Oh, interesting! Well, my client definitely didn't expect that. :P
